<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication2Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>防空反导数字仿真系统平台</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>611</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>目标特性生成</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>参数组1</string>
      </property>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>参数组3</string>
      </property>
      <widget class="QSpinBox" name="spinBox_7">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_8">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_9">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>参数组2</string>
      </property>
      <widget class="QSpinBox" name="spinBox_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_5">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>220</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>参数组4</string>
      </property>
      <widget class="QSpinBox" name="spinBox_10">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_11">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_12">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>510</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>完成设置</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>背景特性生成</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>220</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>参数组4</string>
      </property>
      <widget class="QSpinBox" name="spinBox_13">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_14">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_15">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>参数组2</string>
      </property>
      <widget class="QSpinBox" name="spinBox_16">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_17">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_18">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>参数组3</string>
      </property>
      <widget class="QSpinBox" name="spinBox_19">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_20">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_21">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>参数组1</string>
      </property>
      <widget class="QSpinBox" name="spinBox_22">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_23">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_24">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>干扰特性生成</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>主动雷达回波计算</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>被动雷达回波计算</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>可见/红外回波计算</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>261</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>节点列表</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>231</width>
       <height>581</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>节点选择</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>目标特性生成</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>背景特性生成</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>干扰特性生成</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication2.qrc"/>
 </resources>
 <connections/>
</ui>
